app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comment_count
+    post.increment!(:comments_counter)
+  end
```

Comment:
- Good job done so far, but according to the project requirements, I suggest you call this function after the data has been saved to your database. Please fix this 🙏 i.e `after_save :update_comment_counter`